### ISSUE TITLE
Talking about DD&I -- resources

### ISSUE THREAD

**Issue by sugaroverflow** [drupaldiversity/administration#57]
I'd like to put together resources for talking about DD&I at drupal camps. We can use some of @drnikki or @rubyji's slides as a starting point. It would be good to have examples of how to talk about certain things. 

I think some of the #46 DC Booth talking points apply here.

**Comment by cleverington** [comment]
I'll look when I get a moment, but I think we already have another issue on this topic. If it was in Slack, I remember the conversation gearing towards creating a new 'project' alongside the Event Organizer Packet which we can also publish via GitBook.

**Comment by sugaroverflow** [comment]
@cleverington Hmm, I think I remember this as well. I think this will be a new project/initiative. I'm waiting on this week's meeting on defining the teams and tools and then maybe we can move forward on this :)

**Comment by cleverington** [comment]
Related Post:  (Also where the thought about creating an 'A11Y' packet was broached...... AFTER it was broached in Slack. ;-)

https://github.com/drupaldiversity/administration/issues/44

**Comment by sugaroverflow** [comment]
Closing this as a duplicate of #44  creating onboarding documentation.  Thanks @cleverington !
